runtime/strings/bytes.cpp

```diff
@@ -69,9 +71,10 @@ SortInt hook_BYTES_bytes2int(
   return move_int(result);
 }
 
-unsigned long get_ui(mpz_t i) {
+unsigned long get_ui_named(mpz_t i, std::string caller) {
   if (!mpz_fits_ulong_p(i)) {
-    KLLVM_HOOK_INVALID_ARGUMENT("Integer overflow");
+    KLLVM_HOOK_INVALID_ARGUMENT(
+        "Integer overflow from " + caller + ": ", intToString(i));
```

Comment:
This macro does variadic formatting via a call to `fmt::format`, so we should avoid doing string concatenation to build its argument. See the implementation here: https://github.com/runtimeverification/llvm-backend/blob/571667745aefdcfb4820ba13f3c3f5dcabe12489/include/runtime/fmt_error_handling.h#L6